index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header class="header">
+      <nav class="header_nav">
+        <div class="logo black">
+          <a class="black" href="#">Logo</a>
+        </div>
+        <div class="header_menu">
+          <img class="menu_icon" src="assets/ic_menu.png" alt="menu" />
+        </div>
+      </nav>
+    </header>
+    <section class="headline">
```

Comment:
- [ ] **Required:** This section needs to use flex to align the items inside it, which will also be helpful to make your site responsive for coming parts of your portfolio project

Reply:
I changed my code according to your reviews. My README.md file is completed and I commit and push it to the main branch.